source/onboarding/elasticsearch/index.html.md.erb

```diff
@@ -0,0 +1,21 @@
+---
+title: ElasticSearch
+last_reviewed_on: 2020-03-18
+review_in: 6 months
+weight: 2
+---
+
+# <%= current_page.data.title %>
+
+We have implemented a fix to ElasticSearch to give us more reliability and integrity that the cases in ES will match those in the CCD-Data-Store database.
```

Comment:
It would be good to change the tense a little bit,
This looks more relevant as documentation that was written directly after an issue rather than documentation that is accurate going forward

I think you could write more a long the lines of each team has their own index to improve resiliency...

Reply:
This entire line can go in that case. 

not every team has their own index it's dependent on volume which we'd discuss after they came back with 2.